src/WebUI/Components/PodListComponent.tsx

```diff
@@ -70,11 +71,11 @@ export class PodListComponent extends BaseComponent<IPodListComponentProperties>
         switch (column.key) {
             case podStatusKey:
                 return (
-                    <div className="pod-status">
-                        <Status {...podStatusDic[pod.status.phase]} animated={false} size={StatusSize.s} />
-                        <span className="pod-name"> {pod.metadata.name}</span>
-                    </div>
-                );
+                    <PodStatusComponent 
+                        statusProps={podStatusDic[pod.status.phase]} 
+                        statusSize={StatusSize.m} 

```

Comment:
can we make this as default, and if someone wants to change they can 

 statusSize={StatusSize.m}